### PR TITLE
OVN blog: configure OVN to listen to requests on specified ports

### PR DESCRIPTION
**Pull request by mmirecki**
New OVN configuration introduced in OVS 2.7

Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/blog/2016-11-02-ovirt-provider-ovn.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ Enable the service in order to make it start at boot, use:
 
     systemctl enable ovirt-provider-ovn
 
+OVN central must be configured to listen to requests on appropriate ports:
+
+    ovn-sbctl set-connection ptcp:6642
+    ovn-nbctl set-connection ptcp:6641
+
```
Comment (dankenigsberg) [inline source/blog/2016-11-02-ovirt-provider-ovn.md]: should we not hide this in our own vdsm-tool ovn-configure ?

File: source/blog/2016-11-02-ovirt-provider-ovn.md [hunk 2]
```diff
@@ -119,6 +119,11 @@ Enable the service in order to make it start at boot, use:
 
     systemctl enable ovirt-provider-ovn
 
+OVN central must be configured to listen to requests on appropriate ports:
```
Comment (mkalfon) [inline source/blog/2016-11-02-ovirt-provider-ovn.md]: What happens if you don't configure it? In 2.6, we didn't used it.

File: source/blog/2016-11-02-ovirt-provider-ovn.md (hunk 1 above)
Comment (mmirecki) [inline source/blog/2016-11-02-ovirt-provider-ovn.md]: This is on central, not hosts.
I will add a script for this, also add this to engine-setup.
But I think for now it should be on the blog.

File: source/blog/2016-11-02-ovirt-provider-ovn.md (hunk 2 above)
Comment (mmirecki) [inline source/blog/2016-11-02-ovirt-provider-ovn.md]: Added a note about this being applicable from 2.7

**Pull request merged**